backend/libraries/types/src/user_events.rs

```diff
@@ -10,6 +10,7 @@ pub enum UserEvent {
     ReferredUserRegistered(Box<ReferredUserRegistered>),
     UserSuspended(Box<UserSuspended>),
     OpenChatBotMessage(Box<MessageContent>),
+    UserJoinedGroup(Box<UserJoinedGroup>),
```

Comment:
We should probably define these in the user canister api crate

Reply:
Yeah I actually tried that but the UserJoinedGroup is now part of the LocalUserIndex + UserIndex APIs so it seems nicer leaving it as is.

Reply:
Well I suppose you would define it twice